core-services/prow/02_config/_plugins.yaml

```diff
@@ -697,6 +697,12 @@ config_updater:
         - ci
       gzip: true
       name: ci-operator-misc-configs
+    ci-operator/config/open-cluster-management/**/*-backplane-2.0*.yaml:
```

Comment:
Is there a reason why these can't be `ci-operator/config/open-cluster-management/**/*-backplane-*.yaml` and merged with the one above?

Reply:
Semantically, no - but the org has decided stylistically to explicitly list releases it will process.

Reply:
OK